#36

barkhudaryan-arman opened the pull request:
добавил класс CommentEntity

goodmn79 committed:
# 37 (#44)

barkhudaryan-arman committed:
#36
добавил класс CommentEntity

goodmn79 commented on src/main/java/ru/skypro/homework/dto/comment/CommentEntity.java:
```diff
@@ -0,0 +1,23 @@
+
+import javax.persistence.*;
+@Data
+@NoArgsConstructor
```
- Аннотация лишняя

goodmn79 commented on src/main/java/ru/skypro/homework/dto/comment/CommentEntity.java:
```diff
@@ -0,0 +1,23 @@
+import javax.persistence.*;
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```
- Аннотация лишняя

goodmn79 commented on src/main/java/ru/skypro/homework/dto/comment/CommentEntity.java:
```diff
@@ -0,0 +1,23 @@
+public class CommentEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int pk;
```
- Поле должно иметь тип long, кстати в DTO переименуй тоже)

goodmn79 commented on src/main/java/ru/skypro/homework/dto/comment/CommentEntity.java:
```diff
@@ -0,0 +1,23 @@
+    private int pk;
+    private long author;
+    private String authorImage;
+    private String authorFirstName;
```
- Поля связанные с автором пока убери

goodmn79 commented on src/main/java/ru/skypro/homework/dto/comment/CommentEntity.java:
```diff
@@ -0,0 +1,23 @@
+    private long author;
+    private String authorImage;
+    private String authorFirstName;
+    private int cratedAt;
```
- Поле должно иметь тип long

barkhudaryan-arman committed:
#36
сделал изменения в классе CommentEntity и Comment

# Conflicts:
#	src/main/java/ru/skypro/homework/dto/comment/Comment.java

JuliaCherepneva committed:
#54
Создала класс для валидации данных

JuliaCherepneva committed:
#54
изменила название строки

JuliaCherepneva committed:
#54
изменила метод validateStringLengthRange, который теперь принимает разное количество аргументов

goodmn79 committed:
#54 (#55)

barkhudaryan-arman committed:
Merge remote-tracking branch 'origin/dev' into dev

# Conflicts:
#	src/main/java/ru/skypro/homework/exception/IllegalDataException.java

barkhudaryan-arman committed:
#36
Изменил поля класса CommentEntity
